### ISSUE TITLE
No icons, just puts "null" before any names listed.

### ISSUE THREAD

**Issue by firestrife2** [EwwItsMike/CustomDiscordPresence#1]
Exactly as the title says! Would love to use this plugin.

**Comment by EwwItsMike** [comment]
I had hoped it would work properly before my own clan's bingo started... looked like it did in the test builds when I submitted it too. Quickly came to the realisation it didn't quite work as intended, but the bingo had already started... Hoping to get it fixed soon!

**Comment by firestrife2** [comment]
Suddenly its working? lol

**Comment by EwwItsMike** [comment]
Euh... what? LOL
Tested it just now, and it still removes chatbox messages for me (it doesn't add null like you described for me)

I've just tried a different way to load the images which also seems to work in test environment, and sent the update to runelite devs. Hoping this will fix it for everyone else as well, lmao

**Comment by firestrife2** [comment]
Yeah this morning it is working perfectly. Last night in clanchat it sent "nullRSN" and in friends chat it just made your name disappear. Now it works fine in both

**Comment by firestrife2** [comment]
I would love an option for colorful team icons though - They all kinda blend in with each other XD. Otherwise this looks great!

**Comment by firestrife2** [comment]
If it helps at all, when I built your project in intelliJ, (how you had it before your update 30 mins ago) I could recreate the icons working fine by removing the "/" in the image paths. Instead of "/panelicon.png" just "panelicon.png" and having the images in the main->resources folder. Idk if that makes a difference or helps you at all. Java is not my best language lol

**Comment by firestrife2** [comment]
Just to add its still working perfectly, idk maybe my runelite was bugging yesterday but my whole clan is using it with no issue atm. <3

**Comment by davjet1** [comment]
This issue still exists, it just is fixed by restarting the client. I just ran a bingo with this plugin and restarting clients on anyone who was having issues fixed it.
